hyperspy/learn/rpca.py

```diff
@@ -100,20 +102,16 @@ def svd(X):
 
     # Check options if None
     if lambda1 is None:
-        _logger.warning("Threshold 'lambda1' is set to "
-                        "default: 1 / sqrt(nsamples)")
+        _logger.warning("Threshold 'lambda1' is set to " "default: 1 / sqrt(nsamples)")
```

Comment:
Again with the many "", there are a few examples in this file, and I won't mark them all....  In general you've made quite many 2-line things 1 line in this PR, with no change to content, was that intentional and is everything still under the max. line length?

Reply:
In fact there are quite a few changes e.g. ' to " and regarding adding or removing brackets that look a bit like some editor you've used has made some auto changes? Do we want all of them?

Reply:
resolved most of them